ansible.cfg

```diff
@@ -5,5 +5,3 @@ inventory = ./inventory
 local_tmp = /tmp/ansible
 remote_tmp = /tmp/ansible
 retry_files_save_path = ~/
-callback_plugins = plugins/callback/
-stdout_callback = anstomlog
```

Comment:
Can you remove this callback plugin in a separated commit?

Reply:
:+1: